### DISCUSSION TITLE
How to share a dashboard??

### DISCUSSION THREAD

**Discussion by tradexsrl** [netdata/netdata#17966] [Q&A]
Hi how can i share a custom dashboard created in netdata cloud? the goal will be to present this dashboard to a very large screen for digital signage mounted on a wall to see relevant data but this monitor is not connected to my pc

**Comment by shyamvalsan** [comment]
@tradexsrl Currently you need to login (maybe create a new user with only observer access) from the system the monitor is connected and open the dashboard and make it full-screen. You will NOT be logged out automatically for at least 90 days. (And we can change this timeout on our side if you really need it to be)

We are also working on a TV-mode feature for Netdata which will make what you're trying to do a lot easier.

**Comment by tradexsrl** [comment]
Hi ,thank's for the reply , i'll wait when you release a TV-mode because the url of netdata dashboard will be on a digital signage monitor and i think your login method via magic url by sending an email will not work because a click on the link received in the mail do not open the same browser . i tested this situation on my pc using firefox and chrome , with firefox already logged in , when i open chrome it ask me again to login.
